### PR TITLE
[4.0] Missing bottom border in modal 

### PR DESCRIPTION
**Pull request by NikitaEmberi**
Pull Request for Issue #32190 
### Summary of Changes

Changes as suggested in:https://github.com/joomla/joomla-cms/issues/32190#issuecomment-801452820
and https://github.com/joomla/joomla-cms/pull/32762#issuecomment-803442647

### Actual result BEFORE applying this Pull Request

image

### Expected result AFTER applying this Pull Request

image

### CODE REVIEW

**Comment by NikitaEmberi** [comment]
Please can someone help me how to resolve conflict. I am not really understanding what is wrong.

File: administrator/components/com_associations/tmpl/associations/modal.php [hunk 1]
```diff
@@ -100,8 +100,10 @@
 				?>
 				<tr class="row<?php echo $i % 2; ?>">
 					<?php if (!empty($this->typeSupports['state'])) : ?>
-						<td class="text-center tbody-icon">
-							<span class="<?php echo $iconStates[$this->escape($item->state)]; ?>" aria-hidden="true"></span>
+						<td class="text-center">
+						    <span class="tbody-icon">
```
Comment (Quy) [inline administrator/components/com_associations/tmpl/associations/modal.php]: Please use tabs and not spaces to indent.

File: administrator/templates/atum/scss/blocks/_icons.scss [hunk 2]
```diff
@@ -62,8 +62,8 @@
   &.#{$fa-css-prefix}-star-disabled {
     cursor: not-allowed;
     opacity: 1;
-  }
-
+  } 
+  
   &.disabled .#{$jicon-css-prefix}-home,
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_icons.scss]:
```suggestion
  }

  &.disabled .#{$jicon-css-prefix}-home,
```

**Comment by richard67** [comment]
> Please can someone help me how to resolve conflict. I am not really understanding what is wrong.

Which conflict? I don't see any conflict here. I only see code style errors reported by drone. If you follow the details you end at the log file here: https://ci.joomla.org/joomla/joomla-cms/40993/1/6 .

It mainly says you have to use tabs to indent code.

And which PR is right now? This one or #32786 ? They use different branches but are identical regarding changes. Please close one of these two. There is a button for closing it at the bottom of the PR.

**Comment by richard67** [comment]
Regarding code style you could also apply the suggestions above made by @Quy .

**Comment by NikitaEmberi** [comment]
I'll resolve code style issue and open a new PR. Thank you for letting me know my mistake and I apologize for carelessly opening a PR.

**Comment by drmenzelit** [comment]
You don't need to open a new PR, you can correct your code and make a new commit.

**Review by NikitaEmberi — COMMENTED** [review]
added tab spaces

**Pull request closed without merging**